### ISSUE TITLE
[Engi] Incorrect Armor Mitigation on Turrets on incomming phys. damage

### ISSUE THREAD

**Issue by LestiArdisson** [WarEmu/WarBugs#11731]
I was told by several engineers that the turret seems to be super squishy in the last days, so I checked the damage it takes on an NPC in the Chaos Wastes.

Expected behavior and actual behavior:
Expected behaviour according to the Guide in the RoR Forum:
https://www.returnofreckoning.com/forum/viewtopic.php?f=52&t=18649

Engineer |   |   |  

Gun Turret |   |   |  
  |   |   |   |  
Base   Range: 100ft |   |   |  
Armor: 100% |   |   |  
Resistances: 100% |   |   |  
Hit Points:   50% |   |   |   |  
Weapon   DPS: 70% of Engineer's ranged slot and main hand DPS
  |   |   |   |  
Bombardment Turret |   |  
  |   |   |   |  
Base   Range: 65ft |   |   |  
Armor: 150% |   |   |  
Resistances: 100% |   |   |  
Hit   Points: 75% |   |   |  
Weapon   DPS: 85% of Engineer's ranged slot and main hand DPS
  |   |   |   |  
Flame Turret |   |   |  
  |   |   |   |  
Base   Range: 30ft |   |   |  
Armor: 200% |   |   |  
Resistances: 100% |   |   |  
Hit Points:   100% |   |   |   |  
Weapon   DPS: Engineer's ranged slot and main hand DPS

**Wounds seems to work right:**
According to the guide the Turrets should get the corresponding stats from the casting player.
For example Hit Points: 6920 Hit Points.
Tested on me for total additioning Damage on the Gunturret were 3731 Hitpoints, on the Flameturret 7063 Hitpoints. These were the numbers enemies were putting damage on them so it is a little higher than the real Hitpoints they have. <- This works correct imo.

**Also Resistances seems to work right:**
I have my turrets attack In the east of Chaos Wastes T4 Warcamp, some NPC chaos hounds. 'They have a normal attack and an attack called Bloodthirsty Howl.

My tested damage Numbers gave a 22% Mitigation on that Attack on all 3 Turrets. As my Engis own Resistance Mittigation is 21,6% that seems also to be correct. <- This works correct imo.

**Armor Mitigation seems to be off though:**
My tests show an Armor Mitigation on:
Gunturret = 24% ( 546 damage | 174 mitigation))
Bombard Turret = 36%
Flame Turret = 50% (344 damage | 345 mitigation)

My Engis armor mitigation at the moment of the test was:
Armor = 1910 which is according 43,4 % mitigation. And I don't see this number anywhere corresponding.
The Gunturret should have 100% of my Armor, so also 43,4% mitigation.
The Bombard should have my Armor mitigation *1.5
The Flame Turret should have my Armor mitigation *2

Steps to reproduce the problem:

Go to the chaos wastes and let your different turrets attack something.

The numbers I have collected are in the Excel file.
[Turret Armor Mitigation.xlsx](https://github.com/WarEmu/WarBugs/files/1977285/Turret.Armor.Mitigation.xlsx)

**Comment by grunbag** [comment]
Tested on my engi with 1431 armor :+1: 
my gun turret receiced correctly 1431 armor bonus
my bomb turret receive correctly 2146 armor bonus
and my flame turret received correctly 2862 armor bonus

**Comment by LestiArdisson** [comment]
Hmm.
But my tested NPC Damages on the Turret suggest a different Armor Bonus on the Turrets.
How can you exactly see the Armor Bonus and Mittigation on a Turret ingame without calculating it?

**Comment by grunbag** [comment]
I have GM command that show me my turret stats

**Comment by grunbag** [comment]
Difference might come from toughness or resist I think

**Comment by LestiArdisson** [comment]
Ok I made a retest today on 05.08.2018.
I think I have found an error in my tests and maybe the reason why Pets die so fast on Engi.

It seems like for Turret Armor only the basic Engis Armor is uses. Like Basic armor from Gear, Jewels and Talismans. If I look at that the tested 21% Mittigation on physical damage is correct.

But ofc I made the first test with Armor Buff up "Flak Jacket".
So Lestis mitigation is on 43%.
A resummoned Turret is still on 21%.

So I don't know if this is correct or how it worked in the past.
If in the past the current Armor of the Engi was used, or only his Gear Armor.

Thing is for most engis the Turret is getting killed extreme fast. And that for something like 4-6 months.
